Packs/CortexXDR/Integrations/XQLQueryingEngine/XQLQueryingEngine.py

```diff
@@ -647,7 +653,6 @@ def get_xql_query_results_polling_command(client: Client, args: dict) -> Union[C
         scheduled_command = ScheduledCommand(command='xdr-xql-get-query-results', next_run_in_seconds=interval_in_secs,
                                              args=args, timeout_in_seconds=600)
         command_results.scheduled_command = scheduled_command
-        command_results.readable_output = 'Query is still running, it may take a little while...'
```

Comment:
I removed it because when scheduling a command it looks ugly to have the same repeating message posted to the war room over and over. This removes that message and leaves the user with the spinny time clock thing. It's much prettier.